tests/ubiconfig/utils/test_validation.py

```diff
@@ -1,14 +1,50 @@
 import os
 
 import yaml
+import pytest
 
+from jsonschema.exceptions import ValidationError
 from ubiconfig.utils import config_validation
 
 TEST_DATA = os.path.join(os.path.dirname(__file__),
                          '../../data/configs/dnf7/rhel-atomic-host.yaml')
 
 
-def test_validate_data():
+@pytest.fixture
+def dnf7_config():
     with open(TEST_DATA) as f:
         config = yaml.safe_load(f)
-    assert config_validation.validate_config(config) is None
+    return config
+
+
+def test_validate_data_pass(dnf7_config):
+    assert config_validation.validate_config(dnf7_config) is None
+
+
+def test_validate_data_pass_without_module(dnf7_config):
+    dnf7_config.pop('modules')
+    assert config_validation.validate_config(dnf7_config) is None
+
+
+def test_validate_failed_missing_content_sets(dnf7_config):
+    dnf7_config.pop('content_sets')
+    with pytest.raises(ValidationError):
+        config_validation.validate_config(dnf7_config)
+
+
+def test_validate_failed_extra_keyword(dnf7_config):
+    dnf7_config.update({'extra': 'something should not exist'})
+    with pytest.raises(ValidationError):
+        config_validation.validate_config(dnf7_config)
+
+
+def test_validate_failed_missing_package_include(dnf7_config):
+    dnf7_config['packages'].pop('include')
+    with pytest.raises(ValidationError):
+        config_validation.validate_config(dnf7_config)
+
+
+def test_validate_failed_wrong_data_type(dnf7_config):
+    dnf7_config['packages']['inlcude'] = 'A string'
```

Comment:
I feel like this isn't testing what you wanted, since the test name says "wrong data type" but here you've got a typo 'inlcude'.  Did you mean 'include' so that you'd test what happens if include is a string?

Reply:
yeah, it's a typo here..